SQL/paradise_schema.sql

```diff
@@ -301,6 +301,7 @@ CREATE TABLE `player` (
   `server_region` VARCHAR(32) NULL DEFAULT NULL COLLATE 'utf8mb4_general_ci',
   `muted_adminsounds_ckeys` MEDIUMTEXT NULL DEFAULT NULL COLLATE 'utf8mb4_general_ci',
   `viewrange` VARCHAR(5) NOT NULL DEFAULT '19x15' COLLATE 'utf8mb4_general_ci',
+  `fptp_vote_list` MEDIUMTEXT NULL DEFAULT NULL COLLATE 'utf8mb4_general_ci',
```

Comment:
If this is just for map votes it should probably be named more accordingly

Reply:
Any ideas on what you would want it named as?

Reply:
`map_vote_pref_json`

Reply:
Done